package.json

```diff
@@ -2,10 +2,10 @@
   "name": "js-ecqm-engine",
   "version": "0.0.1",
   "description": "",
-  "main": "lib/executor.js",
+  "main": "lib/index.js",
   "scripts": {
     "test": "jasmine",
-    "lint": "eslint 'lib/*.js' 'lib/**/*.js'"
+    "lint": "eslint 'lib/*.js' 'lib/**/*.js' 'spec/*.js' 'spec/**/.js'"
```

Comment:
Does `spec/**/.js` actually get all the JS files? I feel like it should look more like `spec/**/*.js` to actually lint those files.

Reply:
Pretty sure that's a typo. Changing it now.